apps/web/src/composables/ethers.ts

```diff
@@ -259,23 +250,46 @@ export default function useEthers() {
     getEthersBrowserProviderSelectedCurrency,
     getGasPriceAndLimit,
     loginWithEthers,
+    requestEthersAccount,
     signEthersMessage,
     sendEthersTransaction,
     switchEthersNetwork
   }
 }
 
-function getBrowserProviders(ethereum: any) {
```

Comment:
Refactoring this appears to have resolved the issue I was having (and Lido still appears to have) where attempting to open MetaMask would open CoinbaseWallet instead.

Reply:
Dealing with multiple libraries that pollute the window namespace is difficult. I share your frustration.  What if we created our own object inside window which would include all providers, something like `window.__casimirProviders`. Then each time we load a new provider we would copy its content from window.ethereum to `window.__casimirProviders.etheruem` and delete it from window. This would require us to load the libraries in a different way than we are doing now.
```
window.__casimirProviders = {
    provider1: okx,
    provider2: ethereum,
    provider3: anotherone
}
```